lib/pages/profile_info/profile_info_view.dart

```diff
@@ -0,0 +1,247 @@
+import 'package:fluffychat/utils/clipboard.dart';
+import 'package:fluffychat/utils/string_extension.dart';
+import 'package:fluffychat/utils/twake_snackbar.dart';
+import 'package:fluffychat/widgets/avatar/avatar_style.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:linagora_design_flutter/avatar/round_avatar_style.dart';
+import 'package:linagora_design_flutter/extensions/string_extension.dart';
+import 'package:linagora_design_flutter/linagora_design_flutter.dart';
+
+import 'package:fluffychat/pages/profile_info/profile_info.dart';
+import 'package:fluffychat/pages/profile_info/profile_info_style.dart';
+import 'package:fluffychat/widgets/matrix.dart';
+import 'package:fluffychat/widgets/mxc_image.dart';
+
+class ProfileInfoView extends StatelessWidget {
+  final ProfileInfoController controller;
+  const ProfileInfoView(
+    this.controller, {
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final user = controller.user;
+    final contact = controller.widget.contact;
+    return Scaffold(
+      appBar: AppBar(
+        title: Row(
+          children: [
+            IconButton(
+              onPressed: controller.widget.onBack,
+              icon: const Icon(Icons.close),
+            ),
+            Text(
+              L10n.of(context)!.contactInfo,
+              style: Theme.of(context).textTheme.titleLarge,
+            ),
+          ],
+        ),
```

Comment:
IMO, it did not follow the right dimension 
<img width="419" alt="image" src="https://github.com/linagora/twake-on-matrix/assets/6462404/708c6aa2-368b-40f8-b3fb-be7bfc1fd3aa">

Reply:
Ok, IconButton have default dimension 48x48, I just add small margin for that